README.md

```diff
@@ -1,76 +1,80 @@
-- [Talk to us][13]: Connect to the Cloud Code development team by joining our #cloud-code Slack channel
-- [Prerequisites][10]: A list of prerequisites to use the Cloud Code for VS Code.
-- [Sample Applications][6]: We have starter applications for Node, Python, Go, C# and Java.
-- [File an Issue][7]: If you discover an issue please file a bug and we will fix it as soon as possible.
-- [Request a Feature][8]: If you have any feature requests, ideas for improvement and general feedback please submit a feature request.
-
-[1]: https://github.com/GoogleContainerTools
-[2]: https://skaffold.dev/
-[3]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
-[4]: https://skaffold.dev/docs/how-tos/profiles/
-[5]: https://cloud.google.com/code/docs/vscode
-[6]: https://github.com/GoogleCloudPlatform/cloud-code-samples
-[7]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/new?assignees=&labels=&template=bug_report.md&title=
-[8]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=
-[9]: https://cloud.google.com/code
-[10]: https://cloud.google.com/code/docs/vscode/install
-[11]: https://cloud.google.com/code/docs/vscode/troubleshooting
-[12]: https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/app_deploy_debug.gif
-[13]: https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg
-[14]: https://www.gstatic.com/cloudssh/cloudcode/client-library-browser.gif
-[15]: https://www.gstatic.com/cloudssh/cloudcode/secret-manager-explorer.png
+Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code works with Google’s command-line [container tools](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, and run your applications locally or in the cloud.
+
+![E2E workflow](https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/cloud-code-quick-deploy.gif)
+
+List of Supported Services
+- Kubernetes & Google Kubernetes Engine
+- Cloud Run 
+- Secret Manager
+- Cloud APIs 
+- Cloud Logging 
+- Cloud Build 
+- Kubernetes Config Connector
+- Migrate for Anthos
+
+# Key Features
+
+## Supporting your development workflow
+
+Get run-ready samples, out-of-the-box configuration snippets, support for key Google services like Google Cloud APIs and Cloud Build, one click deployment, a tailored debugging experience, and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!
+
+<details>
+  <summary>Read more</summary>
+  
+  ### Highlights:
+  - Pick your preferred language with Cloud Code’s support for Go, Java, Node.js, Python, .NET Core app development.
```

Comment:
For the time being, add these points under here:

- Get straight to developing with Cloud Code’s simplified authentication workflow that uses your Google Cloud credentials.
- Monitor your app with streaming logs and customize the output with additional filters to produce results meaningful to you.

Reply:
Done.